client/landing/subscriptions/components/add-sites-modal/styles.scss

```diff
@@ -3,7 +3,7 @@
 
 .add-sites-modal {
 	.components-modal__content {
-		margin-top: 60px;
+		min-width: 650px;
```

Comment:
Increased the width of the modal content to ensure the preview doesn't look congested.